src/imputors/standardize.jl

```diff
@@ -0,0 +1,84 @@
+"""
+    Standardize(; values)
```

Comment:
Is ther a name for this that would be less confusing with z-standardizing?

Canonicalize?

Reply:
I don't hate it

Reply:
Opted to make an issue for now cause naming things is hard. I'll probably make these naming issues part of a 0.6 milestone, but I think the smaller PRs will help facilitate discussion.